JS/Lesson27/goksel/editPost/edit-post.css

```diff
@@ -0,0 +1,49 @@
+*{
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    font-family: sans-serif;
+}
+
+#post-title-input {
```

Comment:
Avoid repeating the code, add a class to both input elements and add styles to the class selector.

Reply:
https://en.wikipedia.org/wiki/Don%27t_repeat_yourself